Gemfile.lock

```diff
@@ -161,7 +161,7 @@ GEM
     nokogiri (1.10.3)
       mini_portile2 (~> 2.4.0)
     null_logger (0.0.1)
-    oj (3.8.0)
+    oj (3.7.12)
```

Comment:
Why the downgrade?

Reply:
I honestly don't know I'll check before I merge...

Reply:
Local git fubar so have reverted this